crossterm_terminal/src/terminal/terminal.rs

```diff
@@ -52,18 +52,19 @@ impl Terminal {
     ///
     /// # Example
     /// ```rust
+    /// # use crossterm_terminal::terminal;
     /// let mut term = terminal();
     ///
     /// // clear all cells in terminal.
-    /// term.clear(terminal::ClearType::All);
```

Comment:
I do understeand this change, but I am in a dilema about this. Because if the user used `crossterm` crate, he will see those docs but is not able to use them as `crossterm_terminal` because all types in this crate are re exported in crossterm and therefore they can be: `crossterm`